tools/package-template.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Some of this data I'd like to put in the POM too. Does it make sense to have the package json template be the source of truth? I almost think we could put it in a `.bzl` struct that you can do `json.encode()` on to get this effective file out of. Thoughts?

Reply:
Yeah that works, we can probably defined a `constants.bzl` with a lot of this info

Reply:
Now have a `genrule` to create the base template: https://github.com/player-ui/player/pull/15/files#diff-f354acffd857666c4ea24e57eb1f38b3e9e996ac4a69a3941230c553ec04719eR10